lib/main_containers.pm

```diff
@@ -67,9 +67,9 @@ sub load_host_tests_podman {
 
 sub load_host_tests_docker {
     loadtest 'containers/docker';
-    loadtest 'containers/docker_image';
+    loadtest 'containers/docker_image' unless (is_public_cloud());
```

Comment:
same here, safe to remove `is_public_cloud` here

Reply:
the main problem from my POV is the scheduling in `docker_image` into `load_host_tests_docker`.

Reply:
docker_image is needed also for host tests, cause it's supposed to cover the matrix using released images...

Reply:
But this makes me thing that this module still creates confusion... maybe we should have some split for host and image test...  will think about it.